dracut/modules.d/90kiwi-dump/kiwi-dump-image.sh

```diff
@@ -107,6 +108,26 @@ function get_disk_list {
         fi
         list_items="${list_items} ${disk_device} ${disk_size}"
     done
+    if [ -n "${kiwi_oem_installdevice}" ];then
+        # install device overwritten by cmdline.
+        local device=${kiwi_oem_installdevice}
+        local device_meta
+        local device_size
+        if [ ! -e "${device}" ];then
+            local no_dev="Given device ${device} does not exist"
+            report_and_quit "${no_dev}"
+        fi
+        if [ ! -b "${device}" ];then
+            local no_block_dev="Given device ${device} is not a block special"
+            report_and_quit "${no_block_dev}"
+        fi
+        device_meta=$(
+            eval lsblk "${blk_opts}" "${kiwi_oem_installdevice}" |\
+            grep -E "disk|raid" | tr ' ' ":"
+        )
+        device_size=$(echo "${device_meta}" | cut -f2 -d:)
+        list_items="${device} ${device_size}"
+    fi
```

Comment:
Shouldn't `rd.kiwi.oem.installdevice` override the `kiwi_oemunattended` variable (to true)  as well?

Reply:
Hmm, why should it do that ? If you overwrite the device to install to, how does this affect the "I want to confirm the selected install target interactively yes|no ?" feature. By default kiwi asks before wiping your disk. I think it would be relatively unexpected that it does not do this anymore just because you set rd.kiwi.oem.installdevice right ?

Reply:
I believe it is quite the opposite :) If you are adding the device to the boot parameters you are aware that the device will be written, why would you want to confirm it a second time?

Reply:
ok that actually makes sense to me, you won :-) I agree that someone setting a boot parameter usually should know about the consequences. But let us also add a big fat note about the behavior in the documentation ok ?
I'll adapt the PR according to what you said, thanks much

Reply:
Sounds great, thanks!